app/src/main/res/layout/message_outgoing.xml

```diff
@@ -0,0 +1,25 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/relativeLayout_outgoing"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content">
+
+    <RelativeLayout
```

Comment:
Cirrus says that the RelativeLayout or its parent might be useless, and that we could transfer the background attribute to the other view ?

Reply:
Yes, I tried to fix this one. Same warning was there for message_incoming and I could change that one but for outgoing without the extra layout it jumps back to the left side of the screen so I had to leave it.